src/django/oar/settings.py

```diff
@@ -333,6 +333,9 @@
 DEFAULT_FROM_EMAIL = os.getenv(
     'DEFAULT_FROM_EMAIL', 'noreply@staging.opensupplyhub.org')
 
+DATA_FROM_EMAIL = os.getenv(
```

Comment:
Our `DEFAULT_FROM_EMAIL` is set to an `@staging.opensupplyhub.org'` address, which follows a convention of other configuration settings where the default is staging. Consider continuing that here for consistency.

Reply:
https://github.com/open-apparel-registry/open-apparel-registry/pull/2373/commits/002aeed0a89d032cc14ff6b1310c3b20c2ca3a83